libs/libskynet/src-test/seed.test.ts

```diff
@@ -41,3 +46,50 @@ test("generateSeedPhraseDeterministic", () => {
     expect(found).toBe(true);
   }
 });
+
+// TestMyskyEquivalence is a test that checks that the way libskynet derives
+// the mysky seed for a user matches the code that derived a mysky seed for a
+// user in skynet-mysky. Following the test are some unique dependencies so
+// that the simulated mysky derivation is as close to the original code as
+// possible.
+import nacl from "tweetnacl";
+const SALT_ROOT_DISCOVERABLE_KEY = "root discoverable key";
+function genKeyPairFromSeed(seed: Uint8Array) {
+  const hash = hashWithSalt(seed, SALT_ROOT_DISCOVERABLE_KEY);
+  return genKeyPairFromHash(hash);
+}
+function hashWithSalt(message: Uint8Array, salt: string): Uint8Array {
+  return s512(new Uint8Array([...s512(salt), ...s512(message)]));
+}
+function s512(message: Uint8Array | string): Uint8Array {
+  if (typeof message === "string") {
+    return nacl.hash(stringToUint8ArrayUtf8(message));
+  }
+  return nacl.hash(message);
+}
+function stringToUint8ArrayUtf8(str: string) {
+  return Uint8Array.from(Buffer.from(str, "utf-8"));
+}
+function genKeyPairFromHash(hash: Uint8Array) {
+  const hashBytes = hash.slice(0, 32);
+  const { publicKey, secretKey } = nacl.sign.keyPair.fromSeed(hashBytes);
+  return [publicKey, secretKey];
+}
+test("myskyEquivalence", () => {
+  // Get a seed.
+  let [seedPhrase, errGSPD] = generateSeedPhraseDeterministic("test-for-mysky");
```

Comment:
`const` everywhere.

Reply:
I thought we had a linting rule somewhere that would automatically make everything const?